src/smartStrategy.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+import { UnauthorizedError } from 'fhir-works-on-aws-interface';
+
+export interface AuthStrategy {
+    /**
+     * Validates the JWT access_token
+     * @param accessToken
+     * @throws UnauthorizedError
+     */
+    isTokenValid(accessToken: string): Promise<void>;
```

Comment:
So we are using userinfo to get user identity information perhaps a better method name:
```suggestion
     * @throws UnauthorizedError
     * @returns `fhirUser` claim
     */
    validateToken(accessToken: string): Promise<string>;
```

Reply:
Good suggestion, renamed.